modules/custom/az_paragraphs/css/az_paragraphs_full_width.css

```diff
@@ -4,7 +4,7 @@
 .region-content .full-width-background {
     margin-left: calc(50% - 50vw);
     margin-right: calc(50% - 50vw);
-    max-width: calc(100vw - var(--scrollbar-width) / 2);
+    max-width: calc(100vw - var(--scrollbar-width));
```

Comment:
Why was this removed? It's required to get it to the edge appropriately. Now it doesn't sit flesh with the right edge.

Reply:
Hmmm, it wasn't working, but I can add it back.

Reply:
Yeah that's the limitation I currently have, unable to test on Mac.

On Windows at least...

Here's with the /2:
![edge_with](https://user-images.githubusercontent.com/4400438/142688409-1ec9e102-d5c4-411c-a09b-c1564a2cfb43.png)

And currently without the /2:
![edge_without](https://user-images.githubusercontent.com/4400438/142688520-5c4317fe-c9f3-40e1-9d02-5ddff694812a.png)

It's not terrible, but doesn't achieve full-width.

Reply:
OK, I've reapplied the fix locally, and it looks like it works.

Reply:
Turns out the problem I was having was sidebar implementation specific.  Your solution works for most use cases.